### ISSUE TITLE
NetworkLogger.enabled fires an Unrecognized Selector crash

### ISSUE THREAD

**Issue by scalbatty** [Instabug/Instabug-iOS#273]
### Steps to Reproduce the Problem

Call `NetworkLogger.enabled = false` in swift code

### Expected Behavior

Network logging is disabled

### Actual Behavior

Application crashes with error:
```
+[IBGNetworkLogger setEnabled:]: unrecognized selector sent to class 0x10cab6150
 *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '+[IBGNetworkLogger setEnabled:]: unrecognized selector sent to class 0x10cab6150'
```

### Instabug integration code
*Make sure to remove your app token.*

### SDK Version

3.0.4, integrated through Carthage

### iOS Version

10.3.3 & 12.0 (16A5318d)

### Device Type

iPhone 6 plus & iPhone X

**Comment by Kmohamed** [comment]
I just tried `NetworkLogger.enabled = false` and it works fine are you sure you are on 8.0.4 ?

**Comment by scalbatty** [comment]
OK sorry for the report, I've found the issue and the fix.

We had updated to 8.0.4 from a 7.x version integration and we were still linking `InstabugCore` into the app. Removing it from the app frameworks fixes this issue and #274  as well.

Thank you.

**Comment by Kmohamed** [comment]
No worries If you face anything in migration form 7.x to 8.x do not hesitate to ping me
